mk/config.mk

```diff
@@ -1205,6 +1205,13 @@ $(eval $(call cfg-depends-all,CFG_WIDEVINE_HUK,CFG_DT))
 CFG_WIDEVINE_PTA ?= n
 $(eval $(call cfg-depends-all,CFG_WIDEVINE_PTA,CFG_DT CFG_WIDEVINE_HUK))
 
+# When enabled, CFG_REMOTE_ATTESTATION_PTA embeds remote attestation PTA
+# service.
+CFG_REMOTE_ATTESTATION_PTA ?= n
+ifeq ($(CFG_REMOTE_ATTESTATION_PTA),y)
+$(call force,CFG_QCBOR,y)
+endif
+
```

Comment:
We already have `CFG_ATTESTATION_PTA` so I'm thinking `CFG_REMOTE_ATTESTATION_PTA` may be confusing. Could we come up with a more descriptive name? Should Veraison or CBOR be mentioned perhaps?